internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1106,6 +1106,7 @@ components:
         updated_at: 2020-10-05T12:56:36.362208Z
         version: 2.6.0
         instance_type: standard
+        browser_url: https://console.redhat.com/
```

Comment:
Changed in wrong file. This one is generated by `make openapi/generate`, it should be changed in the openapi/kas-fleet-manager.yaml instead.